pnpm-workspace.yaml

```diff
@@ -10,7 +10,7 @@ catalog:
   drizzle-orm: 0.44.2
   drizzle-kit: 0.31.1
   dd-trace: 5.43.0
-  promptl-ai: 0.8.0
+  promptl-ai: ^0.9.2
```

Comment:
Wait
We need to move promptl to Zod V4. 
https://github.com/latitude-dev/promptl/pull/43

Now is working because is using a promptl version that was deployed with Zod but i reverted it in the code